src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1843,6 +1843,10 @@ bool esbmc_parseoptionst::process_goto_program(
       options.set_option("keep-verified-claims", false);
       options.set_option("no-pointer-check", true);
 
+      // enable '--no-unwinding-assertions' if '--unwind' is enabled
+      if (cmdline.isset("unwind"))
```

Comment:
I think we should enable `--no-unwinding-assertion` only if the user chooses `--branch-coverage`.

@rafaelsamenezes and @mikhailramalho: What are your thoughts here?

Reply:
> I think we should enable --no-unwinding-assertion only if the user chooses --branch-coverage.

I believe that is how it works. We also apply this strategy for assertion and condition coverage as well

```
 if (
      cmdline.isset("assertion-coverage") ||
      cmdline.isset("assertion-coverage-claims"))
{
     if (cmdline.isset("unwind"))
        options.set_option("no-unwinding-assertions", true);
}
```